### PR TITLE
Add ABP configuration to TTI devices

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
#### Summary

Add ABP configuration to avoid break routing when using ABP for EU868 #110 

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/the-things-products/the-things-uno-abp.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: The Things Uno ABP
```
Comment (htdvisser) [inline vendor/the-things-products/the-things-uno-abp.yaml:1]: Shouldn't this just be a different profile for the existing device?

**Review by htdvisser — COMMENTED** [review]
I'm not the expert on these devices (otherwise I'd have done it myself), but this doesn't look quite right to me yet.

**Comment by htdvisser** [comment]
@johanstokking since you contributed quite a lot to these Arduino SDKs I think it's better if you reviewed this.

**Review by johanstokking — CHANGES_REQUESTED** [review]
Agreed with @htdvisser's points:

1. There is no ABP example for The Things Node
2. The Things Uno has an ABP example (`SendABP`: https://github.com/TheThingsNetwork/arduino-device-lib/blob/ea60c59a726d4147aad878a928da5c8baa7faa50/examples/SendABP/SendABP.ino), and that should simply be a firmware version with a profile. This should be under the `the-things-uno` end device
3. RX1 delay is 1 second

**Comment by Jaime-Trinidad** [comment]
I fixed files:

1. I delete ABP on Generic Node.
2. Added a firmware version with a profile in The Things Uno file.
3. Fixed RX1 delay to 1 second.

File: vendor/the-things-products/the-things-uno.yaml [hunk 2]
```diff
@@ -2,6 +2,10 @@ name: The Things Uno
 description: The Things Uno is based on the Arduino Leonardo with an added Microchip LoRaWAN® module. It is fully compatible with the Arduino IDE and existing shields.
 hardwareVersions:
   - version: '1.0'
+    numeric: 1
+  - version: '1.0-ABP'
+    numeric: 2
```
Comment (johanstokking) [inline vendor/the-things-products/the-things-uno.yaml:7]: The hardware is the same

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: vendor/the-things-products/the-things-uno.yaml (hunk 2 above)
Comment (Jaime-Trinidad) [reply vendor/the-things-products/the-things-uno.yaml]: ok, now is fixed, same hardware version

**Review by johanstokking — APPROVED** [review]

**Pull request merged**